docs/learn/learn-proxies.md

```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
 
-- **Step 3**: Go to https://polkadot.js.org/apps/#/extrinsics
-- **Step 4**: Call extrinsic `proxy.killAnonymous` using the selected account ANON PROXY and the following parameters:
+Anon accounts cannot sign anything because they do not have private keys. As a consequence, Anon cannot act as proxies (which is admittedly confusing given their original name), and cannot be included in multi-sig. An Anon account acting as a proxy will not be able to sign anything on behalf of the primary account, and an Anon within a multi-sig will be able neither to initiate nor to approve any call.
 
-  - Spawner: (original account)
-  - Proxy type (kind of proxy)
-  - Index 0 (almost always, but can be seen in creating extrinsic)
-  - Block number x
-  - Extrinsic index y
+:::danger 
 
-  ![call extrinsic](../assets/kill-proxy-3.png)
+Once you remove the relationship with `Any` proxy, the Anon will be inaccessible. Also, Anon accounts cannot sign for anything. For example, do not include Anon in a multi-sig account.
 
-- **Step 5**: Submit and sign extrinsic
+:::
 
-  ![sign extrinsic](../assets/kill-proxy-3.png)
+:::caution Removing anonymous proxies
 
-## How to view your Proxies
+The procedure for removing an Anon is different from the one used to remove proxies. Visit the section "Removing an Anonymous Proxy" on [this support article](https://support.polkadot.network/support/solutions/articles/65000182196), or watch [this technical explainer video](https://www.youtube.com/watch?v=T443RcCYP24).
 
-To view your proxy, head over to the Chain State (underneath "Developer") page on
-[Polkadot-JS Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/chainstate). If
-you've created your proxy on a Kusama account, it is required to change your network accordingly
-using the top left navigation button. On this page, the proxy pallet should be selected, returning
-the announcements and proxies functions. The proxies function will allow you to see your created
-proxies for either one account or for all accounts (using the toggle will enable this). Proxy
-announcements are what time lock proxies do to announce they are going to conduct an action.
+:::
 
-![view proxies](../assets/polkadot_view_proxies.png)
+## Why Anonymous Proxies?
 
-## Putting It All Together
+Despite their complexity and associated dangers, Anon accounts have important benefits. We discuss these benefits below.
```

Comment:
```suggestion
Despite their complexity and associated dangers, `Anon` accounts have important benefits. We discuss these benefits below.
```

Reply:
I remember Ross or Kian saying to put in `` only code stuff. Maybe we can put it in italic? Just trying to stay consistent. We might add these things in the contribution section so that everybody know? Also the title capitalization you sent me yesterday?